common/remote/sockfile.cpp

```diff
@@ -1114,7 +1163,19 @@ protected: friend class CRemoteFileIO;
                 }
             }
             if (!socket) {
-                connectSocket(tep);
+                if (useSSL) {
+                    try {
+                        connectSocket(tep);//first try secure connect
+                    }
+                    catch(...) {
```

Comment:
we tend to only capture IException's (signals are caught and turned into IExceptions)
and then should report exception and release it.